source/administration/important-upgrade-notes.rst

```diff
@@ -8,6 +8,8 @@ Important Upgrade Notes
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
 | v5.14.0                                            | Webhooks are now only displayed if the user is the creator of the webhook or a system administrator.                                                             |
+|                                                    |------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | With the update from Google+ to Google People, system admins need to adjust the ``UserAPIEndpoint`` and need to configure a new Scope configuration setting.     |
```

Comment:
I confirmed this - this setting has been there for a while.  Maybe we can be more explicit? 

With the update from Google+ to Google People, system admins need to ensure the ``GoogleSettings.Scope`` config.json setting is set to ``profile email``and  ``UserAPIEndpoint`` should be set to ``https://people.googleapis.com/v1/people/me?personFields=names,emailAddresses,nicknames,metadata`` per `_updated documentation <https://docs.mattermost.com/deployment/sso-google.html>_.